### ISSUE TITLE
[semantic-issue] Semantics for sklearn RidgeClassifier

### ISSUE THREAD

**Issue by osryde** [spangea/Pyra#176]
**Description**
Invokes the constructor to instantiate a RidgeClassifier object.
Notebook: 0-91-accuracy-score-for-4-models-lr-rc-gbc-rfc.py

**Minimal running example**
```
from sklearn.linear_model import RidgeClassifier

model = RidgeClassifier()
```

**Links to documentation**
https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.RidgeClassifier.html

**Note (optional)**
Returns an object of RidgeClassifier type.
